xarray/core/daskvendor.py

```diff
@@ -0,0 +1,480 @@
+"""Functions vendored from dask."""
```

Comment:
We usually stick these things in `dask_array_compat.py` with the appropriate license. see `pdcompat` and `npcompat`. I guess `dask_compat` would be a better name for these

Reply:
Okay sure, I'm putting it in `daskcompat.py` for consistency with `pdcompat.py` and `npcompat.py` then. 

I feel like "compat" means a different thing in `daskcompat` vs `parallelcompat`, so maybe the latter should be renamed...